src/libraries/System.Runtime/tests/System/Numerics/IBinaryNumberTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Numerics.Tests
+{
+
+    public class IBinaryNumberTests
+    {
+
+        [Fact]
+        public static void AllBitsSetTest()
```

Comment:
nit: Fix newlines and remove unnecessary usings
```suggestion
using Xunit;

namespace System.Numerics.Tests
{
    public sealed class IBinaryNumberTests
    {
        [Fact]
        public static void AllBitsSetTest()
```